ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml

```diff
@@ -31,6 +31,30 @@ tests:
     test:
     - ref: openshift-extended-rosacli-tests
     workflow: rosa-aws-sts-rosacli-e2e
+- as: rosa-aws-non-sts-advanced-f3
+  cron: 23 5 1,4,7,10,13,16,19,22,24,28 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: stable
+      ROSA_LOGIN_ENV: staging
+    workflow: rosa-aws-non-sts-advanced
+- as: rosa-aws-sts-advanced-f3
+  cron: 41 2 2,5,7,11,14,17,20,23,26,29 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: stable
+      ROSA_LOGIN_ENV: staging
+    workflow: rosa-aws-sts-advanced
+- as: rosa-aws-sts-hypershift-advanced-f3
```

Comment:
Could you explain a little bit about why setting all those jobs to f3 ?   CC @jianlinliu

Reply:
These jobs could be used for the daily testing on the cluster provisioning with the rosa cli e2e in the future. Now we enable the day 1 validation testing.

Reply:
Thanks for the explanations.  Waiting for an ack from @jianlinliu as we are exceeding the budgets.

Reply:
ack